index.js

```diff
@@ -80,14 +141,16 @@ module.exports = {
       }
       if (!initSpec(content)) return cb(new Error(initSpec.errorsString))
 
-      const secret = new SecretKey()
-
       ssb.metafeeds.findOrCreate(function gotRoot(err, root) {
         if (err) return cb(err)
 
-        findOrCreateGroupFeed(secret, function gotGroupFeed(err, groupFeed) {
+        findOrCreateGroupFeed(null, function gotGroupFeed(err, groupFeed) {
           if (err) return cb(err)
 
+          console.log('create group feed', groupFeed)
```

Comment:
The `groupFeed` may be empty in case there has been no crash. In that case we should call `findOrCreateGroupFeed(secret, ...`

Reply:
findOrcreateGroupFeed does that internally, see comment 2.2 inside it